src/bin/casr-python.rs

```diff
@@ -0,0 +1,310 @@
+extern crate clap;
+
+use casr::debug;
+use casr::debug::CrashLine;
+use casr::error;
+use casr::execution_class::*;
+use casr::report::CrashReport;
+
+use anyhow::{bail, Context, Result};
+use clap::{App, Arg, ArgGroup};
+use regex::Regex;
+use std::fs::OpenOptions;
+use std::io::Write;
+use std::os::unix::process::ExitStatusExt;
+use std::path::{Path, PathBuf};
+use std::process::Command;
+
+/// Get exception from python report.
+///
+/// # Arguments
+///
+/// * `line` - python exception line
+///
+/// # Return value
+///
+/// ExecutionClass with python exception info
+fn python_exception(line: &str) -> error::Result<ExecutionClass> {
```

Comment:
Unclear parameter name. Does it full report represented as a `&str` or it is just some (which?) line?

Reply:
Done